### ISSUE TITLE
Patchy GroundPrimitive support

### ISSUE THREAD

**Issue by chris-cooper** [AnalyticalGraphicsInc/cesium#4375]
GroundPrimitives have varying levels of support on different devices.  We find on some devices it renders fine, on others with artefacts, on others not at all.  From [the docs](https://cesiumjs.org/Cesium/Build/Documentation/GroundPrimitive.html)...

```
Because of the cutting edge nature of this feature in WebGL, it requires the EXT_frag_depth extension, which is currently only supported in Chrome, Firefox, and Edge. Apple support is expected in iOS 9 and MacOS Safari 9. Android support varies by hardware and IE11 will most likely never support it. You can use webglreport.com to verify support for your hardware.
```

It would be great to have a fallback for lesser devices, particularly given the usefulness for KML etc.

**Comment by mramato** [comment]
Artifacts should definitely be fixed.  Can you please provide some specific video card/OS/browser combinations that have problems?

It's not possible to have a fallback for devices missing the required webgl-extensions (which is unfortunately iOS, IE 11, and some Android developers)  This is a problem that will go away over time as browsers catch up (or in IE 11's case die off).

**Comment by pjcozzi** [comment]
I would be OK with a doc fix here that updates https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Scene/GroundPrimitive.js#L85-L87 to account for the latest iOS/Safari updates.

**Comment by pjcozzi** [comment]
Doc fix in #4888.
